src/libcalamares/PythonHelper.cpp

```diff
@@ -94,9 +98,7 @@ variantListToPyList( const QVariantList& variantList )
 {
     bp::list pyList;
     foreach ( const QVariant& variant, variantList )
-    {
```

Comment:
Both this and the whitespace change were done by astyle (the script in hacking/). We should fix that script :)
I'll correct the Python invocation soon.